app/steps/ui/declaration/index.js

```diff
@@ -205,6 +208,8 @@ module.exports = class Declaration extends ValidationStep {
 
         delete ctx.executorsWrapper;
         delete ctx.hasDataChanged;
+        delete ctx.hasEmailChanged;
+        delete ctx.executorsEmailChanged;
         delete ctx.hasDataChangedAfterEmailSent;
         delete ctx.invitesSent;
         return [ctx, formdata];
```

Comment:
Can you add a unit test for `getContext()` and `action()`.

It doesn't appear that this class has any unit tests so you'll need to create the unit test file...don't worry about unit testing the methods that you've not changed.